dev_guide/expose_service/expose_internal_ip_nodeport.adoc

```diff
@@ -60,6 +60,7 @@ $ oc new-project external-ip
 
 . Edit the service definition to specify `spec.type:NodePort` and optionally specify a port in the in the 30000-32767 range.
```

Comment:
I know this was already there, but would you mind fixing the double "in the in the" here?

Reply:
Absolutely!